OpenSim/Simulation/Model/PrescribedForce.cpp

```diff
@@ -235,34 +257,36 @@ void PrescribedForce::computeForce(const SimTK::State& state,
     const bool hasPointFunctions  = pointFunctions.getSize()==3;
     const bool hasTorqueFunctions = torqueFunctions.getSize()==3;
 
-    assert(_body!=0);
+    const PhysicalFrame& frame =
+        getConnector<PhysicalFrame>("frame").getConnectee();
+    const Ground& gnd = getModel().getGround();
     if (hasForceFunctions) {
         Vec3 force(forceFunctions[0].calcValue(timeAsVector), 
                    forceFunctions[1].calcValue(timeAsVector), 
                    forceFunctions[2].calcValue(timeAsVector));
         if (!forceIsGlobal)
-            engine.transform(state, *_body,                 force, 
-                                    getModel().getGround(), force);
+            force = frame.expressVectorInAnotherFrame(state, force, gnd);
+
```

Comment:
But don't forget the best part - deleting the need for the SimbodyEngine on line 253! :boom: